### PR TITLE
Resolved Item Ownership History issue and error translations in Wallet

### PR DESCRIPTION
**Pull request by faizhashmiRnS**
## Description

Following issues got resolved in this PR:

1. #840 

2. #827

### CODE REVIEW

File: wallet/i18n/de.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "app_title": "Pylonen-Geldbörse",
-  "something_wrong": "Etwas ist schief gelaufen",
+  "something_wrong": "Etwas ist schief gelaufen, Probeer de app opnieuw te starten.",
```
Comment (mustafapylons) [inline wallet/i18n/de.json:3]: This translation has to be fixed, please use German, this is Dutch. i.e. Probieren is German, Probeer is Dutch.

File: wallet/i18n/en.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "app_title": "Pylons Wallet",
-  "something_wrong": "Something went wrong",
+  "something_wrong": "Something went wrong, Try restarting the app.",
```
Comment (mustafapylons) [inline wallet/i18n/en.json:3]: In the Easel fix, this said "Try Restarting the Application", we should be consistent.

File: wallet/i18n/vi.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (mustafapylons) [inline wallet/i18n/vi.json:1]: Please check Vietnamese Translations with Notional Team.

**Review by mustafapylons — COMMENTED** [review]
Couple of issues and Vietnamese translations have to verified by Notional Team.

**Review by faddat — APPROVED** [review]

**Comment by faddat** [comment]
@mustafapylons I am going to enable auto-merge, can you please approve when the changes have been addressed to your satisfaction?

Thanks!

File: wallet/i18n/vi.json (hunk 3 above)
Comment (faizhashmiRnS) [reply wallet/i18n/vi.json:1]: @kjawadDeveloper2 Please coordinate with the Notional team to review Vietnamese Translation.

Comment (faizhashmiRnS) [reply wallet/i18n/vi.json:1]: @kjawadDeveloper2 can we close this issue if its approved and tested?

**Comment by faizhashmiRnS** [comment]
@kjawadDeveloper2 can you please send this pr for qa?

**Review by kjawadDeveloper2 — APPROVED** [review]

**Pull request merged**